installer_scripts/install-yb-voyager

```diff
@@ -879,11 +879,10 @@ centos_install_oracle_instant_clients() {
 
 ubuntu_install_postgres() {
 	output "Installing postgres."
-	line="deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main"
-	sudo grep -qxF "$line" /etc/apt/sources.list.d/pgdg.list || echo "$line" | sudo tee /etc/apt/sources.list.d/pgdg.list 1>&2
-	wget -nv -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+	sudo apt install -y postgresql-common 1>&2
```

Comment:
what all does postgresql-common install?

Reply:
Ah, is it just to be able to run the apt.postgresql.org.sh? 
and that script basically automates the steps like adding to pgdg.list, apt-key add, etc?